spec/controllers/teams_controller_spec.rb

```diff
@@ -42,6 +46,31 @@
     get :index, params: {}
     expect(response).to be_success
   end
+
+  it "should show private teams to team_owner" do
+    team = Team.create! private_attributes
+    team.owners << @user
+    get :index, params: {}
+    teams = controller.instance_variable_get(:@teams)
+    expect(teams.first).to eq(team)
+  end
+
+  it "should not show  private teams to not-logged in user" do
+    team = Team.create! private_attributes
+    team.owners << @user
+    sign_out @user
+    get :index, params: {}
+    teams = controller.instance_variable_get(:@teams)
+    expect(teams).to be_empty
+  end
+
+  it "should not show  private teams to a user who is not part of the team" do
+    team = Team.create! private_attributes
+    team.owners << @other_user
```

Comment:
I would assume you do not need this line of code, because the factory creates a team owner after creating the team. See https://github.com/hpi-swt2/sport-portal/blob/dev/spec/factories/teams.rb.

Reply:
It's not inserting the @other_user though. Might not need it in this place makes it clearer though.